examples/nextjs-comments-overlay/src/components/comments/Comments.tsx

```diff
@@ -3,16 +3,27 @@
 import { ClientSideSuspense } from "@liveblocks/react";
 import { CommentsOverlay } from "@/components/comments/CommentsOverlay";
 import { Toolbar } from "@/components/comments/Toolbar";
+import { ErrorBoundary } from "react-error-boundary";
+import styles from "./Toolbar.module.css";
 
 export function Comments() {
   return (
-    <ClientSideSuspense fallback={null}>
-      {() => (
-        <>
-          <Toolbar />
-          <CommentsOverlay />
-        </>
-      )}
-    </ClientSideSuspense>
+    /* @ts-ignore */
```

Comment:
What was the TS error?

Reply:
`'ErrorBoundary' cannot be used as a JSX component`

Didn't see a way around it, no solutions online worked.